src/component/mxgraph/MxGraphConfigurator.ts

```diff
@@ -58,11 +58,11 @@ export default class MxGraphConfigurator {
     this.graph.foldingEnabled = false;
   }
 
-  private configureMouseNavigationSupport(options?: GlobalOptions): void {
-    const mouseNavigationSupport = options?.mouseNavigationSupport;
+  private configureNavigationSupport(options: GlobalOptions): void {
+    const isNavigationEnabled = options?.navigation?.enabled;
```

Comment:
Do we really need to keep this variable ?

Reply:
No we can remove it

Reply:
done in 9ab9e3bc